l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -37,6 +38,9 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
     /// @dev used to accept the admin role
     address private pendingAdmin;
 
+    /// @notice chainID => baseTokenAssetId
```

Comment:
please describe for the reader the difference between asset id and token. basically so it is more understandable for the first reader the difference between this mapping and the `baseToken` one